src/compiler/transformers/declarations.ts

```diff
@@ -280,33 +283,38 @@ export function transformDeclarations(context: TransformationContext) {
     let errorFallbackNode: Declaration | undefined;
 
     let currentSourceFile: SourceFile;
-    let refs: Map<NodeId, SourceFile>;
-    let libs: Map<string, boolean>;
+    let refs: Set<SourceFile>;
+    let libs: Set<string>;

```

Comment:
Both of these were previously `Map`, but for `libs` only the keys are ever used. And for `refs` I understand form a previous PR that using objects as keys is now preferred (I assume this code was left over from  a time when this was not possible)

If this change is controversial I can revert it